Form/Type/CampaignConditionFieldValueType.php

```diff
@@ -75,7 +75,7 @@ function ($field) {
             ]
         );
 
-        if (isset($fields[$options['data']['field']])) {
+        if (isset($options['data']) && isset($options['data']['field']) && isset($fields[$options['data']['field']])) {
```

Comment:
```suggestion
        if (isset($options['data']['field']) && isset($fields[$options['data']['field']])) {
```
This might be simplified a bit.

Reply:
That's what I thought too, but I was still getting the PHP notice. So I added `isset($options['data'])` and the PHP notice disappeared.

Reply:
That's really strange. I am unable to reproduce it with this simple code snippet.

```php
<?php

declare(strict_types=1);
error_reporting(-1);

$options = [];
$fields = [];

var_dump(isset($options['data']['field']) && isset($fields[$options['data']['field']]));
```

Reply:
We had a zoom with Fedy. It was OPcode cache issue. It can be simplified as Fedy suggests.